src/Router.js

```diff
@@ -25,9 +25,9 @@ class Router {
             handler   : Joi.func().arity(1).required(),
             validation: Joi.object({
                 schema: Joi.object({
-                    isJoi: Joi.valid(true).error(new Error('schema joi be Joi schema only'))
-                }).unknown().default(Joi.any())
-            })
+                    isJoi: Joi.valid(true).error(new Error('schema joi can be Joi schema only'))
+                }).unknown(),
+            }).default({ schema: Joi.any() }),
```

Comment:
no need ',' here
according to this project lint

Reply:
fixed